ARCHITECTURE.md

```diff
@@ -0,0 +1,56 @@
+# PrestoDB: Mission and Architecture
+
+## Introduction
+
+PrestoDB is an open-source distributed SQL query engine designed for fast analytic queries against data sources of all sizes, ranging from gigabytes to petabytes. It was originally developed by Facebook and later made available to the broader community. It is governed under the auspices of the Presto Foundation, a member of the Linux Foundation.
+
+## Mission
+
+### For users
+
+The primary mission of PrestoDB is to enable efficient and high-speed data processing for analytics and batch at scale. It aims to provide a single, unified query system that can access and process data stored in various formats and storage systems. Key aspects of its mission include:
+
+1. **High Performance:** Presto strives to exceed data warehouse performance in data lakes. To achieve this, Presto works to achieve vertical integration over modular portions of the stack.
```

Comment:
The ultimate goal is to avoid needing another tool.  Traditionally, if you had a Data Lake, eventually you'd need a Data Warehouse that you ingest into from the Data Lake, for things like dashboards and analytics.  Do you have any suggestion on how to reword this?

Reply:
Try saying something like that, I suppose. It's much clearer. I completely did not understand that from the original text. maybe,

Presto operates directly on data lakes without first ingesting the tables into a separate data warehouse. 

or something like that?

Reply:
`1. **High Performance:** Presto aims to be a single unified query engine suitable for low latency interactive workloads, adhoc and exploratory analytics, and large scale batch workloads. To achieve this, Presto works to achieve vertical integration over modular portions of the stack.`